examples/flutter_gallery/lib/demo/animation/home.dart

```diff
@@ -200,15 +204,14 @@ class _AllSectionsLayout extends MultiChildLayoutDelegate {
     // When tCollapsed > 0, the indicators move closer together
     //final double rowIndicatorWidth = 48.0 + (1.0 - tCollapsed) * (rowTitleWidth - 48.0);
     const double paddedSectionIndicatorWidth = kSectionIndicatorWidth + 8.0;
-    final double rowIndicatorWidth = paddedSectionIndicatorWidth +
-      (1.0 - tCollapsed) * (rowTitleWidth - paddedSectionIndicatorWidth);
+    final double rowIndicatorWidth =
+        paddedSectionIndicatorWidth + (1.0 - tCollapsed) * (rowTitleWidth - paddedSectionIndicatorWidth);
```

Comment:
I like this transformation, the new version is easier to read. Why isn't the indent two spaces?